db/migrate/20171027162258_create_reviews.rb

```diff
@@ -0,0 +1,11 @@
+class CreateReviews < ActiveRecord::Migration[5.1]
+  def change
+    create_table :reviews do |t|
+      t.integer :rating
+      t.string :body
+      t.references :book, foreign_key: true
```

Comment:
The direct relationship here between book and review indicates to me that a book has many reviews (which is true). You wouldn't see this foreign key on a table that's fully on the opposite side of a join.